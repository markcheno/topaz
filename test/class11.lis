#    0: adjs		1
#    5: newclass	2
#   10: setglobal	0
#   15: pushint		0
#   20: getglobal	0
#   25: pushint		1
#   30: call		new
#   35: pop
#   36: pushint		0
#   41: pushint		10
#   46: pushint		125
#   51: getglobal	0
#   56: pushint		3
#   61: call		setXY
#   66: pop
#   67: pushint		0
#   72: getglobal	0
#   77: pushint		1
#   82: call		show
#   87: pop
#   88: halt
