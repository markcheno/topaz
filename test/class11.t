
p = Point.new
p.setXY(10,125)
p.show
