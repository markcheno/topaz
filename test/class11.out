x=10, y=125
